compute_sdk/globus_compute_sdk/sdk/login_manager/client_login.py

```diff
@@ -34,7 +34,7 @@ def is_client_login() -> bool:
     """
     client_id, client_secret = _get_client_creds_from_env()
 
-    if bool(client_id) ^ bool(client_secret):
+    if client_secret and not client_id:
```

Comment:
Should we update the message below to be more precise?  Or do we not want to highlight that only an ID provided is ok (but that might be for special use cases that might confuse the happy path users?)

Reply:
I think the message gets the point across without over-complicating. If someone only provides a secret, we can be sure that they are trying to do client auth and forgot to specify the client ID.